Sources/SwiftParser/Names.swift

```diff
@@ -261,18 +261,16 @@ extension Parser.Lookahead {
 }
 
 extension Lexer.Lexeme {
-  func canBeArgumentLabel(allowDollarIdentifier: Bool = false) -> Bool {
-    // `inout` is reserved as an argument label for historical reasons.
-    if TypeSpecifier(lexeme: self) == .inout {
-      return false
-    }
-
-    switch self.rawTokenKind {
+  func isArgumentLabel(allowDollarIdentifier: Bool = false) -> Bool {
+    switch self {
     case .identifier, .wildcard:
       // Identifiers, escaped identifiers, and '_' can be argument labels.
       return true
     case .dollarIdentifier:
       return allowDollarIdentifier
+    case .keyword(.inout):
+      // `inout` is reserved as an argument label for historical reasons.
```

Comment:
I realize this is copied from above, but wouldn't this make it *not* reserved, ie. `inout` cannot be an argument label?

Reply:
Yes. Updated